app/views/pages/domains.html.erb

```diff
@@ -55,7 +55,7 @@
     <p class="top_toc"><%=link_to('Back to Top', anchor: '#toc-top') %></p>
 
     <h3 class="about_h3" id="5toc">The domain derivative download</h3>
-    <p class="about_p">By clicking on the "Domains" button, users are provided with a text file that will be named something like 7485-fullurls.txt. You can open this file up in your text editor of choice. By default on MacOS, it will open in TextEdit and look like this:</p>
+    <p class="about_p">By clicking on the "Domains" button, users are provided with a text file that will be named something like 7485-fullurls.csv. You can open this file up in your text editor of choice. By default on MacOS, it will open in TextEdit and look like this:</p>
     <%= image_tag("Tutorial_domain_derivative_file.png", alt: "Screenshot of the domain derivative file in MacOS TextEdit", class:"body_img")%>
```

Comment:
let's swap out the screenshot? If you think this is a good idea I can send it to you via Slack @ruebot (or you can grab it from GitHub below)

alt text: "Screenshot of the domain derivative file in Microsoft Excel"

![Screen Shot 2020-04-16 at 3 29 26 PM](https://user-images.githubusercontent.com/3834704/79498334-1e496e80-7ff7-11ea-888d-e2291cc1a8cf.png)